### DISCUSSION TITLE
How to setup as a local docker container/image

### DISCUSSION THREAD

**Discussion by kortec-elektronik** [INTI-CMNB/KiBot#343] [Other platforms]
Hi there,
is there a documentation or can somebody tell me how to setup KiBot as a docker container/image on my local machine?

**Comment by set-soft** [comment]
Hi @kortec-elektronik !

You can pull already created images, which is explained in the [docs](https://github.com/INTI-CMNB/KiBot/tree/dev#usage-for-cicd).
If you need to create a modified image just start your *Dockerfile* using one of the already created images, i.e.:

```
FROM ghcr.io/inti-cmnb/kicad6_auto:latest
...
```

You can see examples in the root of the repo.

If you want to create very specific and custom images clone the `kicad_auto` [repo](https://github.com/INTI-CMNB/kicad_auto) and modify the Dockerfiles to adjust your needs.

The `kicad_auto` repo contains examples of how to run KiCad or a shell using the images. The [Makefile](https://github.com/INTI-CMNB/KiBot/blob/dev/Makefile) shows how do I run the regression tests for KiCad 5 even when this isn't installed on my system, I use the docker images.

**Comment by kortec-elektronik** [comment]
I got it. Thanks
